src/screens/map-view-screen.tsx

```diff
@@ -151,7 +157,10 @@ export const MapViewScreen = () => {
                             );
                         })}
```

Comment:
Just a small recommendation for syntactic sugar: you do not need to put this `return` here on a map function or the closing `Marker` tag. Instead, do the following:
```ts
product.map(prod => <Marker ...all-your-props... />)
```